### DISCUSSION TITLE
Show information when someone is choosing a beatmap in a multiplayer lobby

### DISCUSSION THREAD

**Discussion by DavidHusicka** [ppy/osu#21799] [Ideas] [answered]
It would be nice if multiplayer lobby showed when someone is choosing a beatmap. This would be useful especially in "queue"-style lobbies where anyone can choose a map. As of now, people quite often end up asking each other whether they are still choosing a map and people announcing that they are going to choose a beatmap. This could be implemented in a similar fashion to information about people downloading maps/being ready with a small text "Choosing a beatmap"

![image](https://user-images.githubusercontent.com/25853767/209560350-e033297f-53d7-4800-9730-c57a8655b759.png)

**Comment by peppy** [comment]
Definitely should be added.

**Comment by frenzibyte** [comment, marked as answer]
Tracked in https://github.com/ppy/osu/issues/11350
